cfgov/legacy/templates/complaint/complaint-landing.html

```diff
@@ -35,13 +35,147 @@
+            <div class="btn-holder">
+              <p><a href="/data-research/research-reports/?title=&categories=consumer-complaint&from_date=&to_date=" class="btn">Read complaint reports</a></p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="block block__border-top block__border-bottom block__padded-top block__padded-bottom block__flush-bottom">
+        <div class="section-two ccdb-landing_section content-l">
+          <div class="content-l_col content-l_col-2-3">
+            <h2 id="about-the-database">About the database</h2>
+            <div class="db-numbers callouts-small">
+              <p class="callout"><span class="numeric-callout percent-timely">97%</span> of complaints sent to companies get timely responses</p>
+            </div>
+            <p>The Consumer Complaint Database is a collection of complaints about consumer financial products and services that we sent to companies for response. Complaints are published after the company responds, confirming a commercial relationship with the consumer, or after 15 days, whichever comes first. Complaints referred to other regulators, such as complaints about depository institutions with less than $10 billion in assets, are not published in the Consumer Complaint Database. The database generally updates daily.</p>
+            <p>
+              <span class="publication-criteria">
+                <a class="a-link a-link__icon" href="https://files.consumerfinance.gov/f/201303_cfpb_Final-Policy-Statement-Disclosure-of-Consumer-Complaint-Data.pdf">
+                  <span class="a-link_text">See publication criteria</span>
+                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 651.7 1200" class="cf-icon-svg"><path d="M507.1 692.8c-15.6-15.6-40.9-15.6-56.6 0l-85.1 85.1V466.6c0-22.1-17.9-40-40-40s-40 17.9-40 40V778l-85.1-85.1c-15.6-15.6-40.9-15.6-56.6 0-15.6 15.6-15.6 40.9 0 56.6L297 902.9c7.5 7.5 17.7 11.7 28.3 11.7s20.8-4.2 28.3-11.7l153.3-153.4c15.8-15.7 15.8-41 .2-56.7z"/><path d="M30 161c-16.5 0-30 13.5-30 30v827.8c0 16.5 13.5 30 30 30h591.7c16.5 0 30-13.5 30-30V343.7L469 161H30zm389.6 60v134.8c0 19.9 16.3 36.2 36.2 36.2h135.9V988.8H60V221h359.6z"/></svg>
+                </a>
+              </span>
+            </p>
+            <p>Have an issue with a financial product or service? We have
+              <a href="/ask-cfpb/">
+                answers to frequently asked questions
+              </a>
+              and can help you connect with companies if you have a complaint.
+              <a href="/complaint/">
+                Learn more
+              </a>
+            </p>
+            <h2 id="what-you-should-consider">What you should consider when using the data</h2>
+            <p>This database is not a statistical sample of consumers’ experiences in the marketplace. Complaints are not necessarily representative of all consumers’ experiences and complaints do not constitute “information” for purposes of the
```

Comment:
Move the text to the next line so it aligns with the link indent.

Reply:
Fixed